Sources/Sentry/SentryLog.m

```diff
@@ -5,16 +5,11 @@
 
 @implementation SentryLog
 
-/**
- * Enable per default to log initialization errors.
- */
-static BOOL isDebug = YES;
-static SentryLevel diagnosticLevel = kSentryLevelError;
+static SentryLevel diagnosticLevel = SENTRY_DEFAULT_LOG_LEVEL;
 static SentryLogOutput *logOutput;
 
 + (void)configure:(BOOL)debug diagnosticLevel:(SentryLevel)level
 {
-    isDebug = debug;
```

Comment:
m: Can we remove the debug parameter too?

Reply:
Good catch 👍🏻